src/edge/acrManager.ts

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import { ContainerRegistryManagementClient } from "azure-arm-containerregistry";
+import { Registry, RegistryListCredentialsResult } from "azure-arm-containerregistry/lib/models";
+import * as request from "request-promise";
+import * as vscode from "vscode";
+import { Executor } from "../common/executor";
+import { UserCancelledError } from "../common/UserCancelledError";
+import { Utility } from "../common/utility";
+import { AcrRegistryQuickPickItem } from "../container/models/AcrRegistryQuickPickItem";
+import { AzureAccount, AzureSession, AzureSubscription } from "../typings/azure-account.api";
+
+export class AcrManager {
+    private readonly azureAccount: AzureAccount;
+    private refreshTokenArc: string;
```

Comment:
Arc or Acr?

Reply:
This name is based on vscode-docker's pattern (for example https://github.com/Microsoft/vscode-docker/blob/master/explorer/models/azureRegistryNodes.ts#L52-L53), as I cannot find ACR's data plane REST API document.